pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -253,7 +253,7 @@ type ServiceClassSpec struct {
 	// ExternalMetadata is a blob of information about the ServiceClass, meant
 	// to be user-facing content and display instructions.  This field may
 	// contain platform-specific conventional values.
-	ExternalMetadata *runtime.RawExtension `json:"externalMetadata, omitempty"`
+	ExternalMetadata *runtime.RawExtension `json:"externalMetadata,omitempty"`
```

Comment:
not the first time I see this. I wonder if we can come up with a versatile solution testing against such bugs.

Reply:
There's a validation script in k8s/k8s that someone just needs to port in.